mgmt_cli_test.py

```diff
@@ -77,6 +78,14 @@ def _get_cluster_hosts_with_ips(self):
             Setup.INTRA_NODE_COMM_PUBLIC else 'private_ip_address'
         return [[node, getattr(node, ip_addr_attr)] for node in self.db_cluster.nodes]
 
+    def get_all_dcs_names(self):
```

Comment:
this is a more general function so it make sense to relocate it under common utils or test-lib or in mgmt.py

it can be used by more tests and this file main purpose is only manager tests.

Reply:
agree, and as part of my plan and need to create tools, i will move part of them to a centralized place once i have enough